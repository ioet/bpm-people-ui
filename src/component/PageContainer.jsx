import React from 'react';
import MenuPanel from './MenuPanel';
import styles from '../generalStyle.css';
import BodyPanel from './BodyPanel';

class PageContainer extends React.Component {
  constructor() {
    super();

    this.state = {
      bodyTitle: 'Nothing',
    };
  }


  render() {
    return (
      <div className={styles.pageContainerStyle}>
        <MenuPanel />
        <BodyPanel />
      </div>

    );
  }
}

export default PageContainer;
