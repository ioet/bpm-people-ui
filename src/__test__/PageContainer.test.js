import React from 'react';
import PageContainer from '../component/PageContainer';

describe('<PageContainer/>', () => {
  test('Renders correctly with children', () => {
    const pageContainerWrapper = shallow(<PageContainer />);

    expect(toJson(pageContainerWrapper)).toMatchSnapshot();

    expect(pageContainerWrapper.find('MenuPanel').length).toEqual(1);
    expect(pageContainerWrapper.find('BodyPanel').length).toEqual(1);
    expect(pageContainerWrapper.find('div.pageContainerStyle').length).toEqual(1);
  });
});
