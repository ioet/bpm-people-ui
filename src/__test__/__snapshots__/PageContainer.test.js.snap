// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageContainer/> Renders correctly with children 1`] = `
<div
  className="pageContainerStyle"
>
  <MenuPanel />
  <BodyPanel />
</div>
`;
